build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -737,6 +843,19 @@ message FileNode {
   string name = 1;
 
   // The digest of the file's content.
+  //
+  // If a virtual filesystem is in use for action execution, e.g. for use with
```

Comment:
>virtual filesystem [](start = 10, length = 18)

Is the notion of virtual filesystem for build known to the Bazel/build community?

Reply:
I enhanced the dynamic command execution doc comments with VFS information to try to improve this situation.

---
In reply to: [459000927](https://github.com/erikma/remote-apis/pull/1#discussion_r459000927) [](ancestors = 459000927)